### ISSUE TITLE
Display issue

### ISSUE THREAD

**Issue by newtoq** [nspire-emus/firebird#129]
For some reason some of the buttons are not displayed correctly. The external LCD does not show up at all.
Win 7 64 bit Intel HD graphics.

![untitled2](https://user-images.githubusercontent.com/24573245/42128881-254b3370-7c7b-11e8-957f-25b9c18a5c0d.png)

![untitled](https://user-images.githubusercontent.com/24573245/42128853-8923af86-7c7a-11e8-9f1c-827f67dfb128.png)

**Comment by Vogtinator** [comment]
Which version? This looks like a broken graphics driver.

**Comment by newtoq** [comment]
Sorry version 1.3 \1.4 \1.4dev. I will reinstall my graphics driver if you think it will help. This is the only program that is giving me display issues. Acer does not believe in supporting driver updates unless it is to fix something that is broken.

**Comment by Vogtinator** [comment]
You can try starting firebird with "QT_OPENGL=desktop" set.

**Comment by newtoq** [comment]
Tried "QT_OPENGL=desktop" set. No dice.

**Comment by newtoq** [comment]
I found a opengl update that made a very big improvement. The display looks good in all versions I have on hand. One minor graphics glitch remains on the ON buton in 1.4. The External LCD still wont appear. It appeared the first time I tried it but not since. Firebird 1.4 dev runs fine until you try to launch a calculator build.Then it crashes. 

![untitled](https://user-images.githubusercontent.com/24573245/42139079-b25c7e0a-7d4c-11e8-8b2e-d17aac9dc437.png)
